website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,490 @@
+---
+title: Customize dbt models database, schema and alias
+id: customize-schema-alias
+description: "Learn how to properly adjust your generate_schema_name() and generate_alias_name() macros."
+displayText: Learn how to adjust your generate schema name and generate alias name.
+hoverSnippet: Learn how to adjust your generate schema name and generate alias name.
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+level: 'Advanced'
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+This guide explains how to customize the [schema](/docs/build/custom-schemas), [database](/docs/build/custom-databases), and [alias](/docs/build/custom-aliases) naming conventions in dbt to fit your data warehouse governance and design needs.
+When we develop dbt models and perform a `dbt run` or `dbt build` command, objects get created in the data warehouse.
```

Comment:
i feel like this section is missing something or not tying to the first sentence at the beginning. maybe something like:
```suggestion
When we develop dbt models and perform a `dbt run` or `dbt build` command, objects (like tables and views) get created in the data warehouse based on these naming conventions.
```